frame/dvm/ethereum/src/lib.rs

```diff
@@ -934,11 +958,19 @@ impl<T: Config> InternalTransactHandler for Pallet<T> {
 		})?
 	}
 
+	/// Execute transaction from other pallets(internal transaction)
+	/// NOTE: The difference between the rpc transaction and the internal transaction is that
+	/// The internal transactions will catch and throw evm error comes from runner to caller.
+	fn internal_transact(target: H160, input: Vec<u8>) -> DispatchResultWithPostInfo {
```

Comment:
This can be removed and merged after helix/wormhole pallets get deleted?

Reply:
Do you mean the `internal_transact` and `read_only_call` can be removed after helix pallets get deleted?

Reply:
I need to check it again.